.changeset/breezy-wasps-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-data": major
```

Comment:
**explanation:** This changes the output of `getGqlRequestId` and so I put this as a major update since that ultimately can effect the behaviour of things that rely on that for cache keys and inflight requests.